src/VoxaEvent.ts

```diff
@@ -24,6 +24,8 @@ export abstract class IVoxaEvent {
   public requestToIntent: ITypeMap = {};
   public requestToRequest: ITypeMap = {};
   public platform!: string;
+  public alexa: any;
```

Comment:
actually, this seems like keys that should only be present in the `AlexaEvent` and `DialogFlowEvent` subclasses

Reply:
Also, they're not `any` we know which keys are present, go ahead and define a type

Reply:
Ok, I'll remove the google one because I'm not using it. I'll just move out the `alexa` key